packages/google-compute-engine/packaging/google-compute-engine-el6.spec

```diff
@@ -0,0 +1,91 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+Name: google-compute-engine
+Version: 2.8.8
+Release 1.el6
+Summary: Google Compute Engine guest environment.
+License: ASL 2.0
+Url: https://github.com/GoogleCloudPlatform/compute-image-packages
+Source0: %{name}_%{version}.orig.tar.gz
+Requires: curl
+Requires: google-compute-engine-oslogin
+Requires: python-google-compute-engine = %{version}
+Requires: rsyslog
+# Old packages
+Obsoletes: google-compute-engine-init
+Obsoletes: google-config
+Obsoletes: google-startup-scripts
+
+BuildArch: noarch
+BuildRequires: systemd
```

Comment:
EL6 doesn't use systemd. Here and below- this should be using the upstart scripts.